packages/less/Gruntfile.js

```diff
@@ -185,10 +185,6 @@ module.exports = function(grunt) {
 
     var path = require('path');
 
-    // Handle async / await in Rollup build for tests
-    const tsNodeRuntime = path.resolve(path.join('node_modules', '.bin', 'ts-node'));
```

Comment:
No.4 npm `workspaces` will hoist all pakcages to the top level, so you can't use these code to exec command line.